ci/code_checks.sh

```diff
@@ -227,7 +227,9 @@ if [[ -z "$CHECK" || "$CHECK" == "doctests" ]]; then
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
     MSG='Doctests generic.py' ; echo $MSG
-    pytest -q --doctest-modules pandas/core/generic.py \
+    pytest -q --doctest-modules \
+        pandas/core/generic.py \
+        pandas/core/groupby/groupby.py \
```

Comment:
You should create a new block instead of adding the file here (the lines from `MSG=...` to `RET=...`).

See the message `Doctests generic.py`, now this become inconsistent. Also the list of functions to skip (the list after the `-k` is not meaningful, as those are being excluded in `groupby.py` where they don't exist.

Reply:
Thank you. I wasn't used to here. Will try to update it accordingly.